lib/node_modules/@stdlib/math/base/special/gamma/manifest.json

```
@@ -0,0 +1,111 @@
+{
+  "options": {
+    "task": "build"
+  },
+  "fields": [
+    {
+      "field": "src",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "include",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "libraries",
+      "resolve": false,
+      "relative": false
+    },
+    {
+      "field": "libpath",
+      "resolve": true,
+      "relative": false
+    }
+  ],
+  "confs": [
+    {
+      "task": "build",
+      "src": [
+        "./src/main.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/math/base/napi/unary",
+        "@stdlib/constants/float64/pinf",
+        "@stdlib/math/base/assert/is-nan",
+        "@stdlib/math/base/assert/is-integer",
+        "@stdlib/math/base/assert/is-negative-zero",
+        "@stdlib/math/base/special/abs",
+        "@stdlib/math/base/special/floor",
+        "@stdlib/math/base/special/sin",
+        "@stdlib/constants/float64/pinf",
+        "@stdlib/constants/float64/ninf",
+        "@stdlib/constants/float64/pi",
+        "@stdlib/constants/float64/eulergamma",
+        "@stdlib/constants/float64/sqrt-two-pi",
+        "@stdlib/math/base/special/pow",
+        "@stdlib/math/base/special/exp"
```

`pinf` is listed twice:

```suggestion
        "@stdlib/math/base/napi/unary",
        "@stdlib/math/base/assert/is-nan",
        "@stdlib/math/base/assert/is-integer",
        "@stdlib/math/base/assert/is-negative-zero",
        "@stdlib/math/base/special/abs",
        "@stdlib/math/base/special/floor",
        "@stdlib/math/base/special/sin",
        "@stdlib/constants/float64/pinf",
        "@stdlib/constants/float64/ninf",
        "@stdlib/constants/float64/pi",
        "@stdlib/constants/float64/eulergamma",
        "@stdlib/constants/float64/sqrt-two-pi",
        "@stdlib/math/base/special/pow",
        "@stdlib/math/base/special/exp"
```